cmd/dao_tests/pubkey_test.go

```diff
@@ -101,7 +102,7 @@ func TestListPubkey(t *testing.T) {
 		t.Errorf("List pubkey test had failed: %v", err)
 		return
 	}
-	assert.Equal(t, 1, len(pubkeys), "List Pubkey error.")
+	assert.Equal(t, 2, len(pubkeys), "List Pubkey error.")
```

Comment:
I have some comments about these tests, many of them do not necessary assert correct things. For example here, the function could store record with all NULLs (that would be incorrect) but this would pass since it only test that "something" was added. I think the correct test is to literally take the listed elements, find the entry and compare it to the inserted entry literally using `==`.

But let's not tackle this in this PR, these tests need more changes.